tokens/sass-helpers.js

```diff
@@ -14,7 +14,7 @@ const chroma = require('chroma-js');
  * @return chroma-js color instance (one of dark or light variants)
  */
 function colorYiq(color, light, dark, threshold) {
-  const defaultsFile = fs.readFileSync(path.resolve(__dirname, 'src', 'global', 'other.json'), 'utf8');
+  const defaultsFile = fs.readFileSync(path.resolve(__dirname, 'src', 'themes/light/global', 'other.json'), 'utf8');
```

Comment:
[question] If/when we have a dark theme variant, I wonder if there is a way we might be able to future proof this `colorYiq` helper function to support multiple theme variants later on (e.g., pulling from `themes/dark/global/other.json` to generate a `dark.css` output)?

The approach we're opting to take here is to have a `light.css` and `dark.css` output file, which would be generated by running the `StyleDictionary.buildAllPlatforms()` twice: once with the light theme variant tokens, and again with the dark theme variant tokens. If there's a way to pass in the current theme variant being used into the `colorYiq` that might be one approach to take here. Open to other ideas of course!

Reply:
Makes sense, I've modified our functions to check whether token belongs to any theme and take theme's colors (defaults to `light`)